core/api_files.go

```diff
@@ -190,3 +190,35 @@ func (a *api) lsThreadFileTargetKeys(g *gin.Context) {
 
 	pbJSON(g, http.StatusOK, keys)
 }
+
+// getFileData godoc
+// @Summary File data at hash
+// @Description Returns raw data for file
+// @Tags files
+// @Produce application/octet-stream
+// @Param hash path string true "file hash"
+// @Success 200 {string} byte
+// @Failure 400 {string} string "Bad Request"
+// @Router /files/{hash} [get]
```

Comment:
file or files? Based on the implementation, seems like this should be `/file/:hash/data`.

Reply:
ah yep, sloppy. all fixed